### PR TITLE
Astro-542 - Favicon Fix and Update

### PR DESCRIPTION
**Pull request by im-cr**
### https://rocketcom.atlassian.net/browse/ASTRO-542

Description:

- Favicon of Astro UXDS updated. Image sizes, asset path links updated for base and home templates.

### CODE REVIEW

**Comment by sknep** [comment]
is this the intended branch? looks like more changes than just a favicon

**Comment by dmcalester** [comment]
This should target `master`

**Comment by sknep** [comment]
This is great! You can totally merge as-is, though if you want to do some image compression, I've found that [ImageOptim](https://imageoptim.com/mac) and [ImageAlpha](https://pngmini.com/) can really shrink the size of PNGs. These aren't really that big, but sometimes it saves as much as 75-80% if the image has only a few colors. 

Definitely not a blocker; just wanted to share the tools in case you're eager to compress the images as much as possible.

File: _includes/base.template.njk [hunk 1]
```diff
@@ -3,7 +3,20 @@
 	<head>
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png">
+		<link rel="icon" type="image/png" sizes="16x16" href="favicon.ico">
```
Comment (sknep) [inline _includes/base.template.njk:6]: Is this path going to the right place? I thought I saw this icon in the img/favicons folder instead. Maybe there's another at the root directory that's already checked in?

File: _includes/home.template.njk [hunk 2]
```diff
@@ -3,7 +3,20 @@
 	<head>
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png">
+		<link rel="icon" type="image/png" sizes="16x16" href="favicon.ico">
```
Comment (sknep) [inline _includes/home.template.njk:6]: same as my other question

**Review by dmcalester — APPROVED** [review]

File: _includes/base.template.njk (hunk 1 above)
Comment (sknep) [reply _includes/base.template.njk:6]: Just wanted to see if you'd had a chance to make sure that the added file at `/img/favicons/favicon.ico` is actually served at `/favicon.ico` through your build process or something :)

**Pull request merged**